modules/virt-adding-templates-to-custom-namespace.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-adding-templates-to-custom-namespace.adoc
+
+[id="virt-adding-templates-to-custom-namespace_{context}"]
+= Adding templates to a custom namespace
+
+The `ssp-operator` deploys Red Hat templates to the `openshift` namespace by default. Templates in the `openshift` namespace are publicly availably to all users. When a custom namespace is created and templates are added to that namespace, virtual machine templates in the `openshift` namespace can be modified or deleted. To add templates to a custom namespace, edit the `hco` custom resource (CR) which contains the `ssp-operator`.
+
+.Procedure
```

Comment:
In this procedure, the user is creating the custom namespace. Therefore, a creating a custom namespace cannot be a prerequisite, because it is being created during the procedure...